README.md

```diff
@@ -6,11 +6,11 @@
 
 ## Composable Botkit Bots
 
-Skellington is a skeleton for your [Botkit](https://github.com/howdyai/botkit) bots. It handles the boilerplate connection 
```

Comment:
Woops, atom.io being aggressive with the whitespace edits there, muddying up the PR like a n00b